spec/javascripts/health-care/components/Nav.spec.jsx

```diff
@@ -57,17 +57,14 @@ describe('<Nav>', () => {
 
     const expectOneActiveLink = (component, path) => {
       history.replace(path);
-      const activeLinks = ReactTestUtils.scryRenderedDOMComponentsWithClass(component, 'usa-current');
+      const activeLinks = ReactTestUtils.scryRenderedDOMComponentsWithClass(component, 'section-current');
       expect(activeLinks).to.have.lengthOf(1);
-      expect(activeLinks[0].getAttribute('href')).to.equal(path);
     };
 
     const expectActiveLinksForNavAndSubNav = (component, path) => {
       history.replace(path);
-      const activeLinks = ReactTestUtils.scryRenderedDOMComponentsWithClass(component, 'usa-current');
+      const activeLinks = ReactTestUtils.scryRenderedDOMComponentsWithClass(component, 'section-current');
       expect(activeLinks).to.have.lengthOf(2);
-      expect(activeLinks[0].getAttribute('href').startsWith(activeLinks[1].getAttribute('href')));
```

Comment:
@awong-dev I would love input on how I can check this since it is not a link anymore without setting this.props.currentUrl as an attribute.